flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/ExecutionSlotSharingGroup.java

```diff
@@ -31,18 +31,19 @@
 import java.util.Set;
 
 /** Represents execution vertices that will run the same shared slot. */
-class ExecutionSlotSharingGroup {
+public class ExecutionSlotSharingGroup {
 
     private final Set<ExecutionVertexID> executionVertexIds;
 
     @Nonnull private final SlotSharingGroup slotSharingGroup;
 
-    ExecutionSlotSharingGroup(@Nonnull SlotSharingGroup slotSharingGroup) {
+    @VisibleForTesting
```

Comment:
VisibleForTesting declares that a function, field, constructor, or entire type is only visible for testing purposes. Therefore, there is no need to add this annotation here.